### PR TITLE
Feature/#342 meta tag

### PR DESCRIPTION
**Pull request by jamesvanmil**
Closes https://github.com/uclibs/scholar_uc/issues/342

### CODE REVIEW

**Comment by hortongn** [comment]
Item 2G in https://scholar.google.com/intl/en/scholar/inclusion.html#indexing says that special characters must be escaped.  Right now, entering the title Testing James' Branch generates a broken meta tag: `<meta name='DC.Title' content='Testing James' Branch'/>`

**Comment by jamesvanmil** [comment]
Good call - I'll work on fixing that. Thanks!

**Comment by hortongn** [comment]
Everything else including the DOI meta tag looks good to me.  Good work!

**Comment by jamesvanmil** [comment]
Fixing the html issue was interesting - learned a bit about html_safe that went contrary to my assumptions about it: http://makandracards.com/makandra/2579-everything-you-know-about-html_safe-is-wrong

TL;DR I used the function escape_once to escape the values of the tags: http://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-escape_once
